exchanges.yml

```diff
@@ -217,17 +217,62 @@ components:
                     type: string
                   template:
                       type: string
-            workflowId:
+            id:
               type: string
-              description: the ID that will be used for the created workflow. workflowId is OPTIONAL.
+              description: The ID that will be used for the created workflow. Passing an ID is OPTIONAL.
     CreateWorkflowResponse:
       type: object
       additionalProperties: false
-      description: Object containing information about a created workflow.
+      description: Response containing the workflowData Object created.
       properties:
-        workflowId:
-          type: string
-          description: The URL that uniquely identifies the created workflow.
+        workflowData:
+          type: object
+          additionalProperties: false
+          description: Object containing information about a created workflow.
+          properties:
+            id:
+              type: string
+              description: The URL that uniquely identifies the created workflow.
+            stepInformation:
+              type: object
+              description: Information about the steps required for the workflow. Returning stepInformation is REQUIRED.
+              properties:
+                exchanges:
+                  type: array
+                  description: The identifiers of the current exchanges associated with the workflow instance.
+                  items:
+                    type: string
+                steps:
+                  type: object
+                  description: One or more steps required to complete an exchange on the workflow.
+                  properties:
+                    stepName:
+                      type: object
+                      description: The name of the step.
+                      properties:
+                        step:
+                          $ref: "#/components/schemas/WorkflowStep"
+                initialStep:
+                  type: string
+                  description: The step from the above set that the exchange starts on.
```

Comment:
I think the `stepInformation` bit should be collapsed to what's in the config, i.e., I think `steps` and `initialStep` are nested here when they shouldn't be. I will have to look at the config section to ensure we don't do that over there too. See note on "current exchanges".

Reply:
This is a copy of the config of an existing workflow returned by the get `workflowId` route, could you provide an example of what the config should look like if this is not it?